wp-content/themes/core/assets/img/admin/blocks/media-text/full.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
I know these are probably just placeholders but should we run these through some kind of minifier? SVGOMG or something?

Reply:
These are strait from our existing panel settings. Very FPO, so I didn't want to worry about optimizing them. I'm putting together a list of the stuff we need from design today yet.